html/mycakeapp/src/Controller/AuctionController.php

```diff
@@ -89,18 +93,18 @@ public function add()
 			// アップロードを許可する拡張子を用意
 			$allowedExtensions = array('jpg', 'jpeg', 'png', 'gif');
 			// アップロードファイルの拡張子取得
-			$fileExtension = pathinfo($_FILES['image']['name'] , PATHINFO_EXTENSION);
-			if(in_array(strtolower($fileExtension), $allowedExtensions, true)){
+			$fileExtension = pathinfo($_FILES['image']['name'], PATHINFO_EXTENSION);
+			if (in_array(strtolower($fileExtension), $allowedExtensions, true)) {
 				// $biditemにフォームの送信内容を反映
 				$biditem = $this->Biditems->patchEntity($biditem, $this->request->getData());
 				// tmp.拡張子で仮置き
-				$biditem['image_path'] = "tmp.".$fileExtension;
+				$biditem['image_path'] = "tmp." . $fileExtension;
 				// $biditemを保存する
-				if($this->Biditems->save($biditem)){
+				if ($this->Biditems->save($biditem)) {
 					// biditem idに名前を変更し拡張子は保持する
 					$image_path = "{$biditem['id']}.{$fileExtension}";
```

Comment:
```suggestion
					$image_path = "{$biditem['id']}.{$fileExtension}";
```
ここでファイル名の先頭にという謎の文字が入るので消してください